torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -1161,10 +1161,19 @@ def clip_grad_norm_(
             self._streams["unshard"],
             self._streams["pre_unshard"],
         )
+        # Check for an early return if every FSDP instance uses `NO_SHARD`
```

Comment:
nit maybe change: Check for an early return if every FSDP instance uses `NO_SHARD` -> "If every FSDP instance uses `NO_SHARD`, early return by calling into the nn.utils helper"?